src/BinaryParsers/ElfBinary/ElfBinary.cs

```diff
@@ -57,13 +57,13 @@ public ElfBinary(Uri uri, string localSymbolDirectories = null, bool forceCompre
                 SectionRegions = ELF.Sections.Where(s => s.LoadAddress > 0).OrderBy(s => s.LoadAddress).ToArray();
 
                 CompilationUnits = new Lazy<List<DwarfCompilationUnit>>(()
-                    => DwarfSymbolProvider.ParseAllCompilationUnits(this,
+                    => LoadDebug(DwarfSymbolProvider.ParseAllCompilationUnits(this,
                                                                     DebugData,
```

Comment:
>DebugData [](http://example.com/codeflow?start=68&length=9)

this argment name (and all others) should be left-aligned with 'this' (the first argument)